src/ert/config/ensemble_config.py

```diff
@@ -172,50 +110,17 @@ def __init__(  # noqa: 501 pylint: disable=too-many-arguments, too-many-branches
         for field in _field_list:
             self.addNode(field)
 
-    @classmethod
-    def _validate_gen_kw_list(cls, gen_kw_list: List[List[str]]) -> None:
-        errors = []
-
-        def find_first_gen_kw_arg(kw_id: str, matching: str) -> Optional[str]:
-            all_arglists = [arglist for arglist in gen_kw_list if arglist[0] == kw_id]
-
-            # Example all_arglists:
-            # [["SIGMA", "sigma.tmpl", "coarse.sigma", "sigma.dist"]]
-            # It is expected to be of length 1
-            if len(all_arglists) > 1:
-                raise ConfigValidationError.with_context(
-                    f"Found two GEN_KW {kw_id} declarations", kw_id
-                )
-
-            return next(
-                (arg for arg in all_arglists[0] if matching.lower() in arg.lower()),
-                None,
+    @staticmethod
+    def _validate(gen_kw_list: List[GenKwConfig], gen_data_list: List[GenDataConfig]):
```

Comment:
This checks only that no duplicate names exist across the gen kws and gen data, then `_validate` suggests it does a bit more than it really does

Reply:
That is the only validation for `EnsembleConfig`, but we could just rename it :)

Reply:
Renamed it :)